JavaAssignments/src/amarja/AllDiffindex.java

```diff
@@ -0,0 +1,35 @@
+package amarja;
+import java.util.Scanner;
+//Question 9 Program to find all different index from given array
+public class AllDiffindex {
+	public static void main(String args[])
+	{		Scanner sc=new Scanner(System.in);//Reading the size and values of array 1
+			System.out.println("Enter size of first array:");
+			int size1=sc.nextInt();
+			System.out.println("Enter values of first array:");
+			int[]num1 = new int[size1];
+			for (int index=0;index<size1;index++)
+			{
+				num1[index]=sc.nextInt();
+			}
+			System.out.println("Enter size of second array:");//Reading the size and values of array 1
+			int size2=sc.nextInt();
+			System.out.println("Enter values of second array:");
+			int[]num2 = new int[size2];
+			for (int index=0;index<size2;index++)
+			{
+				num2[index]=sc.nextInt();
+			}
+			sc.close();
+			AllDiffindex diff=new AllDiffindex();
+			diff.findeqarrind(num1,num2);	//Function to find all different index
+	}
+	void findeqarrind(int num1[],int num2[])
+	{					
+			for (int index=0;index<num1.length;index++)
+				{
+					if (num1[index]!= num2[index])
```

Comment:
We should consider negative scenarios also while writing code. for example what if num1 having more value then num2 . here you are iterating loop till num1.length , if num2 has less elements then comparison num1[index] != num2[index] will throw arrayindex out of bound exception.

Reply:
Amended the code to consider if the array lengths are not equal repeat the loop till the minimum length